assets/i18n/pt-br.json

```diff
@@ -781,6 +781,7 @@
 	"I18N_TOPIC_VIEWER_STUDY_SKILLS_SUBTITLE": "Use os seguintes Cartões de Revisão para ajudá-lo a estudar habilidades sobre <[topicName]>.",
 	"I18N_TOPIC_VIEWER_VIEW_ALL": "Ver tudo",
 	"I18N_TOPIC_VIEWER_VIEW_LESS": "Ver menos",
+	"I18N_TOPIC_iX9kYCjnouWN_TITLE": "Valores do lugar",
```

Comment:
You'll need to drop this prior to submitting the PR.

Reply:
Removed